src/utils/transactions.rs

```diff
@@ -86,6 +109,19 @@ pub(super) fn cancellation_parts_hash(safe_address: &Address, nonce: u64) -> [u8
     keccak256(encoded_parts)
 }
 
+pub(super) fn is_legacy_domain_separator(safe_info: Option<SafeInfo>) -> bool {
+    let version = safe_info
+        .as_ref()
+        .and_then(|safe_info| safe_info.version.as_ref().map(|it| Version::parse(it).ok()))
+        .flatten();
+
+    if let Some(version) = version {
+        version < Version::parse(SAFE_V_1_3_0).unwrap_or(Version::new(1, 3, 0))
```

Comment:
Can we store the parsed versuon in a global constant?

Reply:
I will have to introduce `lazy_static` for this, just FYI

Reply:
Lol it was already there and being used xD